### ISSUE TITLE
CommandBar: InConsistent CommandBar items role.

### ISSUE THREAD

**Issue by bhaveshbhati** [OfficeDev/office-ui-fabric-react#1721]
### Bug Report

- __Package version(s)__: (2.23.3)
- __Browser and OS versions__: (fill this out if relevant)

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (Yes, No)

Requested priority: (Blocking, High, Normal, Low) 

Products/sites affected: (if applicable)

#### Describe the issue:

See this command bar: 

![image](https://cloud.githubusercontent.com/assets/1016447/25699204/dfb75de4-30df-11e7-8de1-145d03ded3f5.png)

Now, Upload and New button has role set to menuitem
![image](https://cloud.githubusercontent.com/assets/1016447/25699240/09a0d3ec-30e0-11e7-8ba6-b45ca9e9210a.png)

But Child items of New like Email message and Calendar event

![image](https://cloud.githubusercontent.com/assets/1016447/25699282/37516702-30e0-11e7-8a97-be5013ce994c.png)

doesn't have the role as menuitem

Is it expected that only parent items button will have role as menuitem ?

#### Actual behavior:

Not all buttons have role as menuitem set

#### Expected behavior:

All buttons should have consistent role ?

### If applicable, please provide a codepen repro:

http://codepen.io/anon/pen/ZKGRBQ

**Comment by betrue-final-final** [comment]
Commands with menus should be sub menus. Just simple commands would be menu items.

Alternately, @cschlechty suggested they all are buttons, and ones with menus use "haspopup", but menu item and sub menu work for me.

**Comment by aadhars** [comment]
@mdahamiwal @micahgodbolt  In current master, the sub menu is also has a role of "menuitem". What is the guidance here. Should submenu items should have role "menuitem" or "submenu" . We marked them as "menuitem" in this PR:
https://github.com/OfficeDev/office-ui-fabric-react/pull/1868

**Comment by cschlechty** [comment]
submenu isn't a valid ARIA role, it should be a menu with menuitems. The button should be a menuitem

**Comment by micahgodbolt** [comment]
I'm guessing that this concern is still valid for the new 6.0 commandbar. But if you haven't tried it yet, please give it a shot. 100% rewrite.

**Comment by kenotron** [comment]
I think the OP might have missed the fact that the in the 2nd example the <li>'s all have menuitem roles already (just not directly in the button). Is this really a bug? Or that the bug is that we need it inside the button instead of li?

**Comment by micahgodbolt** [comment]
@kenotron the OP was talking about the 5.x command bar, so I wanted to make sure our current command bar was actually to spec.

**Comment by kenotron** [comment]
Okay, so I looked at the implementation on https://developer.microsoft.com/en-us/fabric#/components/commandbar

It looks like the BUTTON now has the menuitem role. So, I would think that this can be closed for 6.x branch.
